gprofiler/databricks_client.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+import json
+from typing import Dict, Optional
+
+import requests
+
+from gprofiler.log import get_logger_adapter
+
+HOST_KEY_NAME = "sink.ganglia.host"
+DATABRICKS_DEPLOY_CONF_PATH = "/databricks/common/conf/deploy.conf"
+DATABRICKS_METRICS_PROP_PATH = "/databricks/spark/conf/metrics.properties"
+CLUSTER_TAGS_KEY = "spark.databricks.clusterUsageTags.clusterAllTags"
+JOB_NAME_KEY = "RunName"
+REQUEST_TIMEOUT = 5
+DEFAULT_WEBUI_PORT = 40001
+
+logger = get_logger_adapter(__name__)
+
+
+class DatabricksClient:
+    def __init__(self):
+        self._cluster_deploy_conf_tags: Optional[Dict[str, str]] = None
+        self._is_databricks: bool = False
+        self._is_job: bool = False
+        try:
+            self.job_name = self.get_job_name()
+        except Exception:
+            self.job_name = None
+            logger.warning(
+                "Failed initiating Databricks client. Databricks job name will not be included in "
+                "ephemeral clusters."
+            )
+
+    @property
+    def is_databricks_job(self) -> bool:
+        return self._is_databricks and self._is_job
+
+    def get_webui_address(self) -> Optional[str]:
+        with open(DATABRICKS_METRICS_PROP_PATH) as metrics_properties_file:
```

Comment:
```suggestion
        with open(DATABRICKS_METRICS_PROP_PATH) as f:
```

shorter names for such variables